scripts/contrib-patch-tests.py

```diff
@@ -0,0 +1,94 @@
+"""
+Generate missing integration patch tests.
+
+This scripts scans the ddtrace/contrib folder for all the currently supported
+integrations that implement the patch method in the patch submodule, and checks
+that there is a corresponding patch test is implemented in tests/contrib. It
+then generates a basic patch test for those integrations that do not have one.
+
+Run as
+
+    python scripts/contrib-patch-tests.py
+
+from the repository root folder to automatically (re-)generate missing
+integration patch tests.
+"""
+
+import ast
+from pathlib import Path
+from textwrap import dedent
+
+
+CONTRIB_PATH = Path.cwd() / "ddtrace" / "contrib"
+CONTRIB_TEST_PATH = Path().cwd() / "tests" / "contrib"
+
+
+def _has_unpatch_method(obj):
```

Comment:
Nice! I think we could also generate code like
~~~
try:
    from ddtrace.contrib.aiobotocore.patch import unpatch
except ImportError:
    unpatch = None
~~~